scripts/valhalla_get_elevation

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+set -e
```

Comment:
now we have two elevation scraping scripts. shouldnt we have added the functionality to the python script and kept it to just one elevation script?

Reply:
which Python script? My bash skills are insufficient to handle multiple command line argument streams and switching between the 2 somewhat distinct methods (list of elevation files vs. bounding box).

Reply:
https://github.com/valhalla/valhalla/blob/master/scripts/valhalla_build_elevation

Reply:
no big deal we can reconcile them later